project/sites/judiciaryni/modules/custom/judiciaryni_common/judiciaryni_common.module

```diff
@@ -25,47 +25,70 @@ function judiciaryni_common_entity_presave(EntityInterface $entity) {
   // This will fire when nodes are created or edited.
   if ($entity instanceof NodeInterface && $entity->bundle() === 'judicial_member') {
 
-    $judgeName = $entity->get('field_surname')->value;
-    $judgeSex = $entity->get('field_sex_of_member')->value;
-    $tierTID = $entity->get('field_judicial_tier')->target_id;
-    $judgeTitle = $entity->get('field_prefix_title')->value;
-    $shorttitle = $judgeTitle . ' ' . $judgeName;
-
-    if ($tierTID !== NULL) {
-      $tier = Term::load($tierTID);
-      $tierName = $tier->getName();
-
-      if ($judgeTitle == 'Vacant') {
-        $shorttitle = $judgeName . ' ' . $tierName;
-      }
-      elseif ($tierName == 'Lord Chief Justice' || $tierName == 'Lady Chief Justice') {
-        $shorttitle = $judgeName . ' LCJ';
-      }
-      elseif ($tierName == 'Justice of Appeal') {
-        $shorttitle = $judgeName . ' LJ';
-      }
-      elseif ($tierName == 'High Court Judge') {
-        $shorttitle = $judgeName . ' J';
-      }
-      elseif ($tierName == 'County Court Judge' || $tierName == 'Recorder of Belfast' || $tierName == 'Recorder of Londonderry') {
-        $sex = 'His ';
-        if ($judgeSex !== 'male') {
-          $sex = 'Her ';
-        }
-        $shorttitle = $sex . 'Honour Judge ' . $judgeName;
-      }
-      elseif ($tierName == 'Master of the High Court') {
-        $shorttitle = 'Master ' . $judgeName;
+    $judge_name = $entity->get('field_surname')->value;
+    $judge_sex = $entity->get('field_sex_of_member')->value;
+    $tier_tid = $entity->get('field_judicial_tier')->target_id;
+    $judge_title = $entity->get('field_prefix_title')->value;
+    $short_title = $judge_title . ' ' . $judge_name;
+
+    if ($tier_tid !== NULL) {
```

Comment:
Convention here is usually to wrap the thing being tested for in `empty()` as this tests for all variants of NULL, 0, '0' or '' rather than only the fixed NULL value.

Reply:
`empty()` also improves on `isset()` which carries its own semantic overheads + shortcomings on certain data types